tbarlow12/serverless
# Azure template update

## What did you implement:

- Updated the `serverless.yml` to reference the handler function within the function directories
- Added a second function to show how multiple functions within a Function App would work (common scenario)
- Unified Quickstart docs and README from generated template.
- Added additional documentation to support new structure of template, explained function bindings and instructions for running locally

## How did you implement it:

- See above. Pretty trivial

## How can we verify it:

`sls create -t azure-nodejs -p myApp`

- You'll see two function directories created, `hello` and `goodbye`.
- If you haven't already, `npm install azure-functions-core-tools -g`
- `npm install` - Installs the Azure plugin
- `npm start` - Starts the function app, provides local URLs for both `hello` and `goodbye` functions
- `sls deploy` - Deploys two functions within a function app

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
